### PR TITLE
Additional changes

### PR DESCRIPTION
**Pull request by agnessnowplow**
## Description & motivation
Some leftover changes for the release, see commit name for description.

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -37,6 +37,8 @@ vars:
 ## Upgrading
 Bump the snowplow-web version in your `packages.yml` file.
 
+Some of the tables are relying now on seeds that have just been introduced with this version (`dim_ga4_source_categories`, `dim_geo_country_mapping`,`dim_rfc_5646_language_mapping`). Please run `dbt seed --full-refresh` on your first run.
```
Comment (rlh1994) [inline CHANGELOG:40]: `dbt seed --select snowplow_web --full-refresh`

**Review by rlh1994 — APPROVED** [review]

File: CHANGELOG (hunk 1 above)
Comment (agnessnowplow) [reply CHANGELOG:40]: good point

**Comment by agnessnowplow** [comment]
Ran databricks locally and it failed so I had to replace the vital expected csv.

**Review by rlh1994 — APPROVED** [review]
Looking good, today monday.

**Pull request merged**